packages/dev/babel-preset/index.js

```diff
@@ -55,7 +56,7 @@ module.exports = api => {
               ],
             },
           ],
-          'babel-plugin-minify-dead-code-elimination',
+          require('babel-plugin-minify-dead-code-elimination'),
```

Comment:
These were causing resolution issues when strict/isolated workspace packages were enabled. They now resolve relative to this package and will always use the dependencies of this package first